Description: drop image/avif feature
 "image" in Debian doesn't have feature "avif", so disable it.
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -208,10 +208,10 @@
     "dep:libheif-rs",
     "dep:libheif-sys",
 ]
-libavif = [
-    "image/avif-native",
-    "image/avif",
-]
+# libavif = [
+#     "image/avif-native",
+#     "image/avif",
+# ]
 libraw = ["dep:libraw-rs"]
 
 [target."cfg(windows)".dependencies.file-id]
